paddle/fluid/framework/CMakeLists.txt

```diff
@@ -34,7 +34,9 @@ add_subdirectory(ir)
 add_subdirectory(details)
 # ddim lib
 proto_library(framework_proto SRCS framework.proto)
+proto_library(async_executor_proto SRCS data_feed.proto)
 
+cc_library(util SRCS util.cc DEPS framework_proto)
```

Comment:
util depends on framework_proto however framework_proto is disabled under windows?